test/functional/terminal/tui_spec.lua

```diff
@@ -280,36 +280,30 @@ describe('tui focus event handling', function()
   end)
 
   it('can handle focus events in terminal mode', function()
-    feed_data(':set shell='..nvim_dir..'/shell-test\n')
-    feed_data(':set noshowmode laststatus=0\n')
-
-    retry(2, 3 * screen.timeout, function()
```

Comment:
this was in a retry() because the test was unpredictable. What changed to make it reliable?